app/views/self_service/firms/index.html.erb

```diff
@@ -1,14 +1,21 @@
 <%= render_breadcrumbs(breadcrumbs_root) %>
 
 <div class="l-self-service">
-  <h2><%= t('self_service.firms_index.firm_heading') %></h2>
+  <h1 class="t-page-title">
```

Comment:
This adds in a level 1 page title which was in fact missing from this page. The level 2 is still there below, it was always a local heading to give context to the main firm table.